ietf-layer0-types.yang

```diff
@@ -597,59 +380,16 @@ module ietf-layer0-types {
     }
     reference
       "RFC6205: Generalized Labels for 
-      Lambda-Switch-Capable (LSC) Label Switching Routers";
-  }
-
-  grouping flexi-grid-node-attributes {
-    description "Flexi-grid node attributes";
-    container flexi-grid-node {
-      description "Flexi-grid node attrtibutes";
-      leaf node-type {
-        type identityref {
-          base layer0-node-type;
-        }
-        description "Flexi-grid node type";
-      }
-    }
-    reference
-      "RFC7698: Framework and Requirements for GMPLS-Based Control
-      of Flexi-Grid Dense Wavelength Division Multiplexing (DWDM) 
-      Networks";
-  }
-
-  grouping flexi-grid-path-bandwidth {
-    description "Flexi-grid path bandwidth attributes";
-    leaf bandwidth-type {
-      type identityref {
-        base layer0-bandwidth-type;
-      }
-      description "Flexi-grid bandwidth type";
-    }
-    reference
-      "RFC7698: Framework and Requirements for GMPLS-Based Control
-      of Flexi-Grid Dense Wavelength Division Multiplexing (DWDM) 
-      Networks";
-  }
-
-  grouping flexi-grid-link-bandwidth {
-    description "flexi-grid link bandwidth attributes";
-    leaf-list supported-bandwidth-list {
-      type identityref {
-        base layer0-bandwidth-type;
-      }
-      description "Flexi-grid bandwidth type";
-    }
-    reference
-      "RFC7698: Framework and Requirements for GMPLS-Based Control
-      of Flexi-Grid Dense Wavelength Division Multiplexing (DWDM) 
-      Networks";
+      Lambda-Switch-Capable (LSC) Label Switching Routers
+      ITU-T G.694.2 (12/2003): Spectral grids for WDM applications: 
+      CWDM wavelength grid";
   }
 
   grouping flexi-grid-label-start-end {
     description 
       "Label-start and Label-end information for Flexi-grid.";
     leaf flexi-n {
-      type int16;
+      type l0-types:flexi-n;
       description
         "The central frequency in Flexi-grid.";
```

Comment:
for l0-types:flexi-n the description should be the same as per the typedef of flexi-n that means :"The given value 'N' is used to determine the nominal central frequency." . This is exactly the same what happen for flexi-m leaf, below in the grouping. "M is used to determine the slot width. A slot width is constrained to be M x SWG (that is, M x 12.500 GHz), where M is a positive integer.";